turbo.json

```diff
@@ -77,9 +75,13 @@
     "test": {
       "cache": true,
       "dependsOn": [
-        "build"
+        "build",
+        "^test"
+      ],
+      "outputs": [
```

Comment:
This new config ensures that the following is done before running unit tests:

- The component `build` task is executed
- Unit tests pass for any dependencies

It also invalidates the turborepo cache if any changes have been made to the snapshot directories.